keras/backend/tensorflow_backend.py

```diff
@@ -3052,10 +3054,6 @@ def get_matching_mask(mask_t, ref_tensor_t):
             'maximum_iterations': input_length}
 
         if mask is not None:
-            if not states:
-                raise ValueError('No initial states provided! '
```

Comment:
This is a change of behavior, so this should be approved by @fchollet.

Reply:
Sure, I'd say it is just a leftover from the previous PR, see [this line](https://github.com/keras-team/keras/pull/11499/files?utf8=%E2%9C%93&diff=unified#diff-a18b8c6a1191d6f49303e0d599ca8c37L3081). This requirement is now a bit arbitrary and is not there in the other backends.

Reply:
Thanks for giving the context.